solutions/bluegreen/pipelines/pipeline.json

```diff
@@ -0,0 +1,108 @@
+{
+	"name": "bluegreen",
+	"id": "bluegreen",
+	"application": "helloworld",
+	"appConfig": {},
+	"description": "blue green pipeline",
+	"executionEngine": "v2",
+	"keepWaitingPipelines": false,
+	"lastModifiedBy": "anonymous",
+	"limitConcurrent": true,
+	"parallel": true,
+	"stages": [{
+			"account": "my-kubernetes-account",
+			"cloudProvider": "kubernetes",
+			"manifestArtifactAccount": "embedded-artifact",
+			"manifests": [{
+				"apiVersion": "apps/v1",
+				"kind": "ReplicaSet",
+				"metadata": {
+					"annotations": {
+						"strategy.spinnaker.io/max-version-history": "2",
+						"traffic.spinnaker.io/load-balancers": "[\"service hellosvc\"]"
+					},
+					"labels": {
+						"tier": "helloworld"
+					},
+					"name": "helloworld",
+					"namespace": "default"
+				},
+				"spec": {
+					"replicas": 3,
+					"selector": {
+						"matchLabels": {
+							"tier": "helloworld"
+						}
+					},
+					"template": {
+						"metadata": {
+							"labels": {
+								"app": "helloworld",
+								"tier": "helloworld"
+							}
+						},
+						"spec": {
+							"containers": [{
+								"image": "gcr.io/spinnaker-marketplace/helloworld:v2",
+								"imagePullPolicy": "IfNotPresent",
+								"name": "helloworld",
+								"ports": [{
+									"containerPort": 6000
+								}]
+							}]
+						}
+					}
+				}
+			}],
+			"moniker": {
+				"app": "helloworld"
+			},
+			"name": "Deploy (Manifest)",
+			"refId": "1",
+			"relationships": {
+				"loadBalancers": [],
+				"securityGroups": []
+			},
+			"requisiteStageRefIds": [],
+			"source": "text",
+			"type": "deployManifest"
+		},
+		{
+			"account": "my-kubernetes-account",
+			"app": "helloworld",
+			"cloudProvider": "kubernetes",
+			"cluster": "replicaSet helloworld",
+			"criteria": "second_newest",
+			"kind": "replicaSet",
+			"location": "default",
+			"manifestName": " ",
+			"mode": "dynamic",
+			"name": "Disable (Manifest)",
+			"refId": "2",
+			"requisiteStageRefIds": [
+				"1"
+			],
+			"type": "disableManifest"
+		},
+		{
+			"account": "my-kubernetes-account",
+			"app": "helloworld",
+			"cloudProvider": "kubernetes",
+			"cluster": "replicaSet helloworld",
+			"criteria": "second_newest",
+			"kind": "replicaSet",
+			"location": "default",
+			"manifestName": " ",
+			"mode": "dynamic",
+			"name": "Scale (Manifest)",
+			"refId": "3",
+			"replicas": 1,
+			"requisiteStageRefIds": [
+				"2"
+			],
+			"type": "scaleManifest"
+		}
+	],
+	"triggers": [],
+	"updateTs": "1531749445000"
```

Comment:
I think this can be deleted, right @jtk54?

Reply:
Yes.

Reply:
Yes.